lint/private/maybe.bzl

```diff
@@ -0,0 +1,7 @@
+"maybe utilities"
```

Comment:
I really don't like this pattern, what other "maybe utilities" are there? I think it was better when this two-line function was inlined at the use site

Reply:
It ends up being 4 lines since there are two load statements vs. one line to just load the maybe. My preference is the one line since it seems wasteful to copy the same 4 lines everywhere you need the maybe.

To answer your question of "what other maybe utilities", some repos need `maybe_http_file` as well and I've seen a few other variants. A `private/maybe.bzl` is a home for that boilerplate so use sites still need just one line if there are mutiple variants vs 4 lines X <number of variants>.

Reply:
bzlmod will make all this go away in the near future one way or the other

Reply:
unrelated to maybe, can you tell why this PR CI is red for the one e2e test???